src/tests/Microsoft.PowerFx.Interpreter.Tests/ValueTests.cs

```diff
@@ -156,8 +156,6 @@ private class RowDontMarshal
             private int PrivateProp { get; set; }
 
             internal int InternalProp { get; set; }
-
-            public int publicField;
```

Comment:
This is an intended part of the test, to show that it isn't marshalled. If there's a warning, you can use the appropriate #pragma  to disable it

Reply:
Restored.